react-app-render/src/Main.tsx

```diff
@@ -1,67 +1,65 @@
+const resourceUri = "/public/Samples/Reports/9.CustomerDetailReport";
 const getLiveSamplePage = (pageName: string, vContainer: any) => {
-    switch (pageName){
-        case 'inputControls':
-            return <InputControlsLiveSample vContainer={vContainer}/>
-        case 'inputControlsTypes':
-            return <InputControlTypes vContainer={vContainer}/>
-        case 'scheduler':
-                return <SchedulerLiveSample vContainer={vContainer} uri={resourceUri}/>
-        case 'application':
-            return <App visualize={vContainer} uri={resourceUri}/>
-        default:
-            return <></>
-
-    }
-}
+  switch (pageName) {
+    case "inputControls":
+      return <InputControlsLiveSample vContainer={vContainer} />;
+    case "inputControlsTypes":
+      return <InputControlTypes vContainer={vContainer} />;
+    case "scheduler":
+      return <SchedulerLiveSample vContainer={vContainer} uri={resourceUri} />;
+    case "application":
+      return <App visualize={vContainer} uri={resourceUri} />;
+    default:
+      return <></>;
+  }
+};
 
 const Main = () => {
-    const vContainer = useVisualize(visualizeUrl);
-    const [sampleComponent, setSampleComponent] = useState<ReactNode | null>(null)
-
+  const vContainer = useVisualize(visualizeUrl);
+  const [sampleComponent, setSampleComponent] = useState<ReactNode | null>(
+    null,
+  );
 
-    useEffect(() => {
-        if(vContainer) {
-            const lastPartOfUrl = window.location.href;
-            let pageName = ""
-            if(lastPartOfUrl.endsWith(inputControlsPageStart)){
-                pageName = 'inputControls'
-            } else if (lastPartOfUrl.endsWith(inputControlsTypesPageStart)){
-                pageName = 'inputControlsTypes';
-            } else if(lastPartOfUrl.endsWith(schedulerPageStart)){
-                pageName = 'scheduler'
-            } else if(lastPartOfUrl.endsWith(applicationPageStart)) {
-                pageName = 'application'
-            }
-            setSampleComponent(getLiveSamplePage(pageName, vContainer))
-        }
-
-    }, [vContainer])
-    return (
-        <>
-            {
-                vContainer ? sampleComponent : <>Loading...</>
-            }
-
-        </>
-    );
-}
+  useEffect(() => {
+    if (vContainer) {
+      const lastPartOfUrl = window.location.href;
+      let pageName = "";
+      if (lastPartOfUrl.endsWith(inputControlsPageStart)) {
+        pageName = "inputControls";
+      } else if (lastPartOfUrl.endsWith(inputControlsTypesPageStart)) {
+        pageName = "inputControlsTypes";
+      } else if (lastPartOfUrl.endsWith(schedulerPageStart)) {
+        pageName = "scheduler";
+      } else if (lastPartOfUrl.endsWith(applicationPageStart)) {
+        pageName = "application";
+      } else if (lastPartOfUrl.endsWith("input-controls/report-sample.html")) {
+        setSampleComponent(
+          <ReportSample vContainer={vContainer} uri={resourceUri} />,
```

Comment:
move this to `getLiveSamplePage()` method, to have the same behavior please.

Reply:
I propose we eliminate `getLiveSamplePage` from all samples, as it adds unneeded complexity to our samples by adding a new layer of mapping from URL -> sample component

Reply:
ok sounds good, pls remove it for all options.